apps/explorer-e2e/src/integration/validator.cy.js

```diff
@@ -87,7 +87,7 @@ context('Validator page', { tags: '@smoke' }, function () {
         });
     });
 
-    it('should be able to see relevant node information in vega data section', function () {
+    xit('should be able to see relevant node information in vega data section', function () {
```

Comment:
question (non-blocking): if this test is breaking, i think leaving a comment would help for future reference explaining what's going on